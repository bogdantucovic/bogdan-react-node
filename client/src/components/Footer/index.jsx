import React from 'react';

import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';
import withStyles from '@material-ui/core/styles/withStyles';
import styles from './index.style';
import StyledLink from '../UIcomponents/StyledLink';

const Footer = ({ classes }) => (
  <footer 
    className={classes.footer}
  >
    <Grid container
      justify="space-around" 
      alignItems="center"
    >
      <StyledLink
        href="https://www.linkedin.com/in/btucovic/" 
        target="_blank"
        rel="author"
      >
        Linkedin
      </StyledLink>
      <Typography variant="caption" >
        Developed by Bogdan Tucovic
      </Typography>
      <StyledLink
        href="https://github.com/bogdantucovic" 
        target="_blank"
        rel="author"
      >
        GitHub
      </StyledLink>
    </Grid> 
  </footer>
)

export default withStyles(styles)(Footer);