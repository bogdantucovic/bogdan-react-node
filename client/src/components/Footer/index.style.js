export default theme => ({
  footer: {
    backgroundColor: theme.palette.primary.contrastText,
    borderTopColor: theme.palette.primary.main,
    borderTopStyle: 'dotted',
    borderTopWidth: '1px'
  }
});