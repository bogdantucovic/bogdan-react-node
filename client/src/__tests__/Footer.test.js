import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../components/Footer/';

describe('<Footer/>', () => {
  const wrapper = shallow(<Footer/>);

  it('render', () => {
    expect(wrapper).toMatchSnapshot()
  });
});