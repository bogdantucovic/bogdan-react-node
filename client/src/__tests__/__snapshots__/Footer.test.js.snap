// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> render 1`] = `
<footer>
  <WithStyles(AppBar)
    color="inherit"
    position="static"
  >
    <WithStyles(Toolbar)>
      <WithStyles(Grid)
        alignItems="center"
        container={true}
        justify="space-around"
      >
        <WithStyles(StyledLink)
          href="https://www.linkedin.com/in/btucovic/"
          rel="author"
          target="_blank"
        >
          Linkedin
        </WithStyles(StyledLink)>
        <WithStyles(Typography)
          variant="caption"
        >
          Developed by Bogdan Tucovic
        </WithStyles(Typography)>
        <WithStyles(StyledLink)
          href="https://github.com/bogdantucovic"
          rel="author"
          target="_blank"
        >
          GitHub
        </WithStyles(StyledLink)>
      </WithStyles(Grid)>
    </WithStyles(Toolbar)>
  </WithStyles(AppBar)>
</footer>
`;
